install/kubernetes/cilium/templates/clustermesh-apiserver/deployment.yaml

```diff
@@ -291,8 +299,11 @@ spec:
         {{- with .Values.clustermesh.apiserver.kvstoremesh.extraEnv }}
         {{- toYaml . | trim | nindent 8 }}
         {{- end }}
-        {{- if .Values.clustermesh.apiserver.metrics.kvstoremesh.enabled }}
```

Comment:
is this `if` now in the wrong place?

Reply:
I don't think so. The `kvmesh-health` port is used for readiness checks and should be included regardless of whether metrics are enabled or not. This conditional is now just for adding the metrics port to the list.